api/interservice/compliance/reporting/reporting.proto

```diff
@@ -17,6 +17,7 @@ service ReportingService {
 	rpc ReadNode(Id) returns (Node) {};
 	rpc ListNodes(Query) returns (Nodes) {};
 	rpc ListControlItems(ControlItemRequest) returns (ControlItems) {};
+	rpc ReadNodeInfo(Query) returns (NodeReport) {};
```

Comment:
change the method name as GetNodeHeaderInfo, input parameter as Id, and output parameter as NodeHeaderInfo

Reply:
Done